app/models/task-list.js

```diff
@@ -0,0 +1,11 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+import { belongsTo, hasMany } from 'ember-data/relationships';
+
+export default Model.extend({
+  name: attr('string'),
+  position: attr('number'),
```

Comment:
We may want to wait on naming convention until this settles in https://github.com/code-corps/code-corps-api/pull/557

@green-arrow are discussing `rank` vs `order`.

One thing that is missing here, though, is the distinction, which will require some manual work in the mirage routes for the task. We'll need to make it so that `position` can be set on the model, but it's a virtual attribute on the server that then gets turned into `rank` or `order`, which is a completely different number.

This is a little odd but setting and getting are not the same thing, or even the same property.